docfx_yaml/extension.py

```diff
@@ -1002,6 +1031,43 @@ def pretty_package_name(package_group):
     return " ".join(capitalized_name)
 
 
+# Given generated markdown files, incorporate them into the docfx_yaml output.
+# The markdown file metadata will be added to top level of the TOC.
+def find_markdown_pages(app, outdir):
+    # Use this to ignore markdown files that are unnecessary.
+    files_to_ignore = [
+        "index.md",     # merge index.md and README.md and index.yaml later
+        "reference.md", # Reference docs overlap with Overview. Will try and incorporate this in later.
+        "readme.md",    # README does not seem to work in cloud site
```

Comment:
Correct. The file would need to have a different name.

Reply:
Gotcha, I'll look into what to name this. Filing an issue.